samples/unit-tests/data/csv/demo.js

```diff
@@ -713,3 +685,27 @@ QUnit.test('Dot date format', function (assert) {
         }
     });
 });
+
+QUnit.test(
+    'Hex values are not treated as comments (#13283).',
+    function (assert) {
+        var chart = Highcharts.chart('container', {
```

Comment:
Instead of creating a full `Chart` instance, we can test this using the Data module only. See some of the other tests in the same file for how we call `Highcharts.data` and do the assertion in the `parsed` callback.